cluster/charts/crossplane/README.md

```diff
@@ -61,7 +61,7 @@ Install with Helm 2:
 
 ```console
 helm repo add crossplane-master https://charts.crossplane.io/master/
-helm search crossplane-master --devel
```

Comment:
I am fairly certain more recent versions of helm 2 do include `--devel`. I recently updated the integration tests to use `--devel` and I believe they use helm 2.

Reply:
If you do not specify it then it uses `alpha`

Reply:
https://v2.helm.sh/docs/helm/#options-8

Reply:
> If you do not specify it then it uses alpha

I think that happens when you run `helm search crossplane` but here we're explicitly looking in `crossplane-master`. Running helm 2.15.1 `helm search crossplane-master` gives me:
```
NAME                                    	CHART VERSION         	APP VERSION           	DESCRIPTION
crossplane-master/crossplane            	0.11.0-rc.146.g7d91391	0.11.0-rc.146.g7d91391	Crossplane is an open source Kubernetes add-on that exten...
crossplane-master/crossplane-controllers	0.11.0-rc.146.g7d91391	0.11.0-rc.146.g7d91391	Crossplane - Managed Cloud Resources Operator
crossplane-master/crossplane-types      	0.11.0-rc.146.g7d91391	0.11.0-rc.146.g7d91391	Crossplane - Managed Cloud Resources Operator
crossplane-master/gitlab-controller     	0.0.0-43.90e2a0e      	0.0.0-43.90e2a0e      	GitLab Controller - GitLab Crossplane Application
```

Reply:
Gotcha! Thanks for clarifying :)